app/src/main/java/com/github/sdp/ratemyepfl/fragment/review/ReviewListFragment.kt

```diff
@@ -9,28 +8,19 @@ import androidx.recyclerview.widget.DividerItemDecoration
 import androidx.recyclerview.widget.RecyclerView
 import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
 import com.github.sdp.ratemyepfl.R
-import com.github.sdp.ratemyepfl.activity.AddReviewActivity
 import com.github.sdp.ratemyepfl.adapter.ReviewAdapter
-import com.github.sdp.ratemyepfl.model.items.Reviewable
-import com.github.sdp.ratemyepfl.utils.ListActivityUtils
 import com.github.sdp.ratemyepfl.viewmodel.ReviewViewModel
-import com.google.android.material.floatingactionbutton.ExtendedFloatingActionButton
 
 open class ReviewListFragment : Fragment(R.layout.fragment_review_list) {
     private lateinit var reviewsAdapter: ReviewAdapter
     private lateinit var recyclerView: RecyclerView
-
-    private lateinit var fab: ExtendedFloatingActionButton
```

Comment:
I guess you move this functionality elsewhere, I'll check it.

Reply:
Yes the button is in the navigation now, if you still want a button in the list, we can add it back.